docker/docker_client.go

```diff
@@ -96,6 +94,10 @@ type dockerClient struct {
 	scheme             string // Empty value also used to indicate detectProperties() has not yet succeeded.
 	challenges         []challenge
 	supportsSignatures bool
+	// The tlsClientConfig is setup during the creation of the dockerClient and
+	// will be updated by detectPropertiesHelper(). Any HTTP request the
+	// dockerClient does will be done by this TLS client configuration.
+	tlsClientConfig *tls.Config
```

Comment:
(Absolutely non-blocking: This would be better placed above the “// The following members are detected registry properties:” section, so that the order of members in here follows the order of use: created in constructor, set after calling constructor, set based on all of the above, (private do not touch).)

Reply:
Fix proposed in https://github.com/containers/image/pull/617 .